### PR TITLE
Jekoch/fix comment

### PR DESCRIPTION
**Pull request by Jessica-Koch**
specify that submit closes a dialog

### CODE REVIEW

File: packages/teams-js/src/public/dialog.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export namespace dialog {
    * Hide from docs because this function is only used during initialization
    * ------------------
    * Adds register handlers for messageForChild upon initialization and only in the tasks FrameContext. {@link FrameContexts.task}
-   * Function is called during app intitialization
+   * Function is called during app initialization
```
Comment (erinha) [inline packages/teams-js/src/public/dialog.ts:44]: ❤️

File: packages/teams-js/src/public/dialog.ts [hunk 2]
```diff
@@ -95,7 +95,7 @@ export namespace dialog {
   }
 
   /**
-   * Submit the dialog module.
+   * Submit the dialog module and closes the dialog
```
Comment (erinha) [inline packages/teams-js/src/public/dialog.ts:98]: This should be "and close the dialog" to match tenses

File: change/@microsoft-teams-js-0dc95483-5ebc-4037-a819-ab2a4968825e.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "add code comment for dialog",
```
Comment (erinha) [inline change/@microsoft-teams-js-0dc95483-5ebc-4037-a819-ab2a4968825e.json:3]: Please follow the guidance for change files (found here: https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md#change-log-using-beachball).

In this case, I suggest (note that 'dialog.submit' is encased in backticks that GitHub auto-formats):

Added clarifying comment to `dialog.submit` to indicate the dialog is closed when `submit` is called.

**Review by erinha — CHANGES_REQUESTED** [review]
Please update the change file comment and tense of updated comment. Thanks for fixing the other typo!

File: packages/teams-js/src/public/dialog.ts (hunk 1 above)
Comment (Jessica-Koch) [reply packages/teams-js/src/public/dialog.ts:44]: I got a spell checker! :) it's great

File: packages/teams-js/src/public/dialog.ts (hunk 2 above)
Comment (Jessica-Koch) [reply packages/teams-js/src/public/dialog.ts:98]: you're right!

**Review by erinha — APPROVED** [review]

**Review by TrevorJoelHarris — APPROVED** [review]

**Pull request merged**